.makenodebabelrc

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": ["env"],
+  "plugins": [
+    "transform-class-properties",
+    "transform-flow-strip-types",
+    [
+      "transform-inline-environment-variables",
+      {
+        "include": [
+          "PACKAGE_VERSION"
+        ]
+      }
+    ]
+  ]
+}
```

Comment:
can you elaborate why this cannot be the default .babelrc and you need to copy it back and forth?

Reply:
@yurishkuro
When running tests in Node 0.10, the `babel-plugin-transform-inline-environment-variables` cannot be used because it is only compatible with Node 6+. Additionally, this rule cannot be defined as a cli argument because the `includes` option must be defined otherwise different environment variables are replaced with `undefined`.
Is there somewhere in the codebase I should add this comment?

Reply:
> When running tests in Node 0.10

Tests with 0.10 are run against transpiled code, at which point the replacement should have happened already.

Reply:
The tests themselves also need to transpile at test execution time. This plugin doesn't work in 0.10 so it is used at build time (Node 6), and not used the rest of the time (variable node version).

Reply:
so why go through all these hoops then?  Your earliest solution of just writing `version.js` file during build was much simpler imo. Simple unix commands instead of plugin always get my vote.

Reply:
I leaned towards the plugin to avoid making a file in `dist` off of a file in `src`, but since the file is phenomenally simple (export a single string) it does seem easier than maintaining two `.bablerc` files or figuring out why a `babel.config.js` file isn't working.

I reverted the branch back to the previous, simpler state.